src/components/MovieCard/MovieCard.tsx

```diff
@@ -28,14 +31,25 @@ export const MovieCard: React.FC<Props> = ({ movie }) => (
           </figure>
         </div>
         <div className="media-content">
-          <p className="title is-8" data-cy="movieTitle">{movie.title}</p>
+          <p
+            className="title is-8"
+            data-cy="movieTitle"
+          >
+            {movie.title}
+          </p>
```

Comment:
if the row length is less than max allowable size of row, and has less than 3 attrs  - no need to move each attr on the new row